tee-worker/ts-tests/integration-tests/common/utils/identity-helper.ts

```diff
@@ -128,32 +128,6 @@ export async function buildValidations(
             'LitentryValidationData',
             bitcoinValidationData
         );
-        validation = encodedVerifyIdentityValidation;
-    } else if (network === 'bitcoinPrettified') {
```

Comment:
So, removing this means: Prettified messages are supported for `TrustedCall` and `Getter` signatures.

This code was testing a prettified message on `ValidationData`, which clients don't prettify.

It is also a consequence of moving the prettify signature check from `LitentryMultiSignature` to `TrustedCall`/ `Getter`.

`ts-tests` doesn't really cover prettified messages or blake256 payloads. All messages are raw. I'm leaving the ts-test rework to https://linear.app/litentry/issue/P-639/clean-up-remove-raw-payload-signature-verification

cc @Kailai-Wang – this was the missing fix for the CI checks.

Reply:
> IDHub know about it

I think IDHub never knew challenge codes could be prettified 😂

But to be fair, it hasn't been a high-priority issue. Now that I am touching this msg prefix I'll bring up the discussion if we want prettified challenge codes too.